build/root/usr/local/bin/ci_entrypoint_sro.sh

```diff
@@ -34,6 +33,17 @@ test_master_branch() {
     validate_sro_deployment
 }
 
+test_custom_image() {
+    CUSTOM_IMAGE="$1"
+    CI_IMAGE_SRO_COMMIT_CI_REPO="${2:-https://github.com/openshift-psap/special-resource-operator.git}"
+    CI_IMAGE_SRO_COMMIT_CI_REF="${3:-master}"
```

Comment:
I think @omertuc woud usually suggests to use something like that:
```
    CUSTOM_IMAGE="$1"
    shift || true
    CI_IMAGE_SRO_COMMIT_CI_REPO="${1:-https://github.com/openshift-psap/special-resource-operator.git}"
    shift || true
    CI_IMAGE_SRO_COMMIT_CI_REF="${1:-master}"
```

Reply:
I kept the file's style, but I can do this change if needed.

Reply:
yes, we usually try to focus on having *new* code following our informally-defined common best practise, so if you want to update this block as I suggested ... and perhaps the rest of the file for consistency :)

Reply:
Done in e2a143808b08741d5d5604d3f8524c6dd547205f